usr/share/rear/finalize/DP/default/500_restore_ssc.sh

```diff
@@ -13,8 +13,8 @@
 test -s /etc/opt/omni/client/ssconfig || return 0
 
 local sscpath=/etc/opt/omni/client/sscertificates
-local certfile=$sscpath/localhost_cert.pem
-local keyfile=$sscpath/localhost_key.pem
+local certfile=$sscpath/localhost_cert.*
```

Comment:
@sebastian-koehler are you sure that the wildcard will match one certfile and not two?

Reply:
Hello @gdha, the file name is either localhost_cert.pem for older versions or localhost_cert.enc for more recent versions of Data Protector.